KnowSheet/scripts/check-headers.sh

```diff
@@ -10,28 +10,30 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
 LDFLAGS="-pthread"
 
-TMPDIR=.noshit
+# WARNING: TMP_DIR must be resolved from the current working directory.
```

Comment:
Useful comment, but, IMHO, it doesn't really deserve TO BE FULLY CAPITALZIZED :)